connection.js

```diff
@@ -918,9 +918,12 @@ class Connection {
                     `${this.local.host} Hello ${this.get_remote('host')}${this.ehlo_hello_message}`,
                     "PIPELINING",
                     "8BITMIME",
-                    "SMTPUTF8",
                 ];
 
+                if (!!this.cfg.main.smtputf8) {
```

Comment:
```suggestion
                if (this.cfg.main.smtputf8) {
```

Reply:
I'm new to JavaScript, won't this throw is this.cfg.main.smtputf8 is undef?

Reply:
Nope. JS isn't strongly typed. In a case like this, undefined is the same as "not true."